src/changes-backwards-compatible.apib

```diff
@@ -5,7 +5,9 @@ We list all backwards-compatible additions here. These are currently available i
 (There is also a [list of backwards-incompatible upgrades](#changelog) available, but those only apply if you [upgrade your API version](#upgrading-your-api-version).)
 
 #### May 2023
+
 - We added `team_lead_id` to the filters on `teams.list`.
+- We added `sort` to `users.list`.
```

Comment:
I think we can be more explicit in the sorting here.

Maybe a good idea to combine all the sorting additions into one sentence.

```suggestion
- We added sorting on `first_name` and `last_name` to `users.list`.
```